docs/src/pages/docs/crocks/Pred.md

```diff
@@ -78,6 +78,42 @@ filter(largeNumber, [ 200, 375, 15 ])
 
 </article>
 
+<article id="topic-construction">
+
+## Construction
+
+```haskell
+Pred :: (a -> Boolean) -> Pred a
+```
+
+The `Pred` constructor is a unary function that requires a unary predicate
+function as its sole argument. Once provided with its predicate function,
+the constructor will return a new `Pred` instance wrapping the predicate.
+
+If the function provided is not a predicate, then the resulting value will be
+coerced to a `Boolean`.
+
+```javascript
+import Pred from 'crocks/Pred'
+
+import and from 'crocks/logic/and'
+import isArray from 'crocks/predicates/isArray'
+import not from 'crocks/logic/not'
+
+// isEmpty :: a -> Boolean
+const isEmpty =
+  not(x => !!x.length)
```

Comment:
This is super trvial but am i missing something or could this not just be?

const isEmpty =
  x => !x.length

Reply:
I do too, so many people hate it but i think it's such a beautiful part of JS